docs/data/joy/main-features/color-inversion/ColorInversionNavigation.js

```diff
@@ -131,28 +137,27 @@ export default function ColorInversionNavigation() {
         </Card>
       </Sheet>
       <Sheet
-        variant="soft"
-        color="neutral"
+        variant="solid"
         invertedColors
-        sx={(theme) => ({
+        sx={{
           p: 2,
           display: 'flex',
           flexDirection: 'column',
+          alignItems: 'center',
           gap: 2,
-          ...(color !== 'neutral' && {
-            bgcolor: `${color}.900`,
-          }),
+          bgcolor: `${color}.900`,
+
+          '& .MuiBadge-root': { '--Badge-ringColor': 'unset' },
+          '& .MuiBadge-colorSuccess': { bgcolor: 'success.400' },
           '& button': {
             borderRadius: '50%',
             padding: 0,
-            '&:hover': {
-              boxShadow: theme.shadow.md,
-            },
+            '--IconButton-size': '3rem',
           },
-        })}
+        }}
       >
-        <Badge badgeContent="7" size="sm">
-          <IconButton>
+        <Badge badgeContent="7" badgeInset="15%" size="sm">
```

Comment:
```suggestion
        <Badge badgeContent="7" badgeInset="10%" size="sm">
```

Pushing it up just a bit more to the side.